src/components/Comments/Comment.js

```diff
@@ -38,6 +46,7 @@ const Comment = ({
   onLongPress,
   updateVote
 }) => {
+  const refSheet = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `refSheet` using `React.useRef()` is a good practice when dealing with mutable values that may change over time without causing re-renders. This seems to be used later for controlling the visibility of the bottom sheet menu.

```diff
+ const refSheet = React.useRef();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->